tutorials/core-concepts/pieces/pieces.md

```diff
@@ -0,0 +1,64 @@
+---
```

Comment:
If I come straight to this page from a google search I won't know what a piece is. That should be recapped.